plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -736,6 +775,54 @@ func isValidCustomFields(customFields []string) bool {
 	return true
 }
 
+func getCertIdHash(reqData requestData, zone string, logger hclog.Logger) string {
+	logger.Debug("Creating hash for certificate ID")
+	s := ""
+	if reqData.commonName != "" {
+		s = s + reqData.commonName + ";"
+	}
+
+	// unless it's a common name, we want the sans to be separated by comma
+	if len(reqData.altNames) > 0 {
+		orderSANDNS(&reqData, logger)
+		for index, altName := range reqData.altNames {
+			if index == len(reqData.altNames)-1 {
+				s = s + altName
+			}
+			s = s + altName + ","
+		}
+	}
+
+	s = s + ";" + zone
+
+	s = sha1sum(s)
+	return s
+}
+
+func addCNtoSANDNS(reqData *requestData, logger hclog.Logger) {
```

Comment:
Do function name really needs the SAN part to be understood?
`addCNtoDNS` or `addCNtoDNSList` reads better and gives the same context, DNS is part of SANs